includes/polling/wifi.inc.php

```diff
@@ -16,6 +16,11 @@
         include 'includes/polling/mib/siklu-mib.inc.php';
     }
 
+    if ($device['os'] == 'saf') {
+        echo "It is SAF\n";
```

Comment:
I just followed what had been written before on the echo. There is a number of if's and all them work exactly in the same way, but i agree we should change them to what you suggest, and probably do else if's instead of just a ton of if's considering a device is never gonna be 5 different brands. What do you think?